src/components/HeartFavorite/HeartFavorite.jsx

```diff
@@ -4,17 +4,17 @@ import propTypes from 'prop-types';
 import { useSelector, useDispatch } from 'react-redux';
 import swal from 'sweetalert';
 
-import { addToFavorites, removeFromFavorites } from '../../store/actions';
-import { addFavorite, removeFavorite } from '../../Favorites/favorites';
+import { addFavorite, removeFavorite } from 'store/actions/favoritesActions';
+import { addFavorite as addFavoriteToDatabase, removeFavorite as removeFromDatabase } from '../../Favorites/favorites';
```

Comment:
it looks weird that from `store/actions/favoritesActions` the linter changes it to be relative and this does not change...